test/dotcom_web/controllers/app_store_controller_test.exs

```diff
@@ -0,0 +1,63 @@
+defmodule DotcomWeb.AppStoreControllerTest do
+  use DotcomWeb.ConnCase, async: true
+  import Test.Support.EnvHelpers
+
+  test "redirects to default project page by default", %{conn: conn} do
+    reassign_env(:dotcom, :mbta_go_app, default_project_page: "/default_project_page")
+
+    conn =
+      get(
+        conn,
+        app_store_path(conn, :redirect_mbta_go, %{
+          "pt" => "fake pt",
+          "ct" => "fake ct",
+          "mt" => "fake-mt"
+        })
+      )
+
+    assert redirected_to(conn, 302) =~ "/default_project_page"
+  end
+
+  test "redirects to app store for ios browser", %{conn: conn} do
+    reassign_env(:dotcom, :mbta_go_app, ios_store_base_url: "fake_app_store_url?")
```

Comment:
suggestion: we tend to use `Faker` to generate fake values and avoid having the "magic value" issue. `Faker.Internet.url()` can give you a random URL and random values can be used for the param values too.

Reply:
Oh cool! Opted to move the values out of config anyway, but good tip!